### ISSUE TITLE
[Bug]: BluetoothLE MacOS / iOS Not Scanning when Not Set Up for Background Bluetooth

### ISSUE THREAD

**Issue by TPitman-mindfiretechnology** [shinyorg/shiny#1216]
### Component/Nuget

BluetoothLE Client (Shiny.BluetoothLE)

### What operating system(s) are effected?

- [X] iOS (13+ supported)
- [X] Mac Catalyst
- [ ] Android (8+ supported)

### Version(s) of Operation Systems

MacOS: 12.6
iOS: 16.2

### What platform(s) are effected?

- [X] MAUI
- [ ] Native
- [ ] Classic Xamarin
- [ ] Other

### Steps To Reproduce

1. Create maui project and get shiny nuget libraries.
2. Scan for peripherals (sample code shows how I scan).

I created a test project as required.

It should currently be set so that it is using the nuget packages. If I run it as it and hit the button, it doesn't detect any bluetooth devices even though there are several around me.

If I remove the nuget reference and add the Shiny.Bluetooth project from the latest source code and make the changes mentioned in this discussion topic:

[https://github.com/orgs/shinyorg/discussions/1215](url)

Then it does scan and find devices both on MacOS and iOS.

### Expected Behavior

Returns scan results on MacOS, iOS and Android

### Actual Behavior

Only returns scan results on Android

### Exception or Log output

none

### Code Sample

https://github.com/TPitman-mindfiretechnology/ShinyBLETest.git

### Code of Conduct

- [X] I have supplied a reproducible sample is NOT FROM THE SHINY SAMPLES!
- [X] I am a Sponsor OR I am using the LATEST stable/beta version from nuget (v2.7.1 stable - ALPHAS are not taking issues)
- [X] I understand that if I am checking these boxes and I am not actually following what they are saying, I will be removed from this repository!

**Comment by aritchie** [comment]
The iOS platform folder is missing.  I need the info.plist settings

**Comment by TPitman-mindfiretechnology** [comment]
Dang. Must be one of the gitignore settings. I thought all of that was in there. Stay tuned....

**Comment by TPitman-mindfiretechnology** [comment]
Sorry about that. I do a lot of Unity 3D projects and usually put the builds in a folder called ios, which I had ignored in gitignore. The tool I used for that didn't put the / in front of it so it ignored all ios folders.

Ok, git project updated. By the way I did put the required strings in both the ios and MacCatalyst folders. The project does run on MacCatalyst and doesn't work without my manual changes to the source code, but does work with them.
